### PR TITLE
Trigger sanityCheck for all member PRs

### PR DESCRIPTION
**Pull request by blindpirate**
### Context

This makes attempt to resolve https://github.com/gradle/gradle-private/issues/3009

### CODE REVIEW

File: .teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt [hunk 1]
```diff
@@ -52,6 +55,19 @@ fun BuildFeatures.publishBuildStatusToGithub(model: CIBuildModel) {
...
 
+fun BuildFeatures.triggeredOnPullRequests() {
+    pullRequests {
+        vcsRootExtId = "Gradle_Branches_GradlePersonalBranches"
+        provider = github {
+            authType = token {
+                token = "credentialsJSON:5306bfc7-041e-46e8-8d61-1d49424e7b04"
+            }
+            filterAuthorRole = PullRequests.GitHubRoleFilter.MEMBER
+            filterTargetBranch = masterReleaseBranchFilter
```
Comment (jjohannes) [inline .teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt]: This looks wrong as this should run the branch of the PR? (Maybe no filter is needed here at all?)

Comment (blindpirate) [reply .teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt]: It's "target branch filter", but yes, we can of source use "all branch" here. According to [the doc](https://www.jetbrains.com/help/teamcity/branch-filter.html), all branch filter is 

```
+:*
```

Comment (blindpirate) [reply .teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt]: Fixed, PTAL!

**Comment by blindpirate** [comment]
Thanks @jjohannes !

**Pull request merged**